src/schema/search.ts

```diff
@@ -455,7 +467,20 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
                 query: `%${query}%`,
               });
           }),
-        )
+        );
+      if (includeContentPreference) {
+        searchQuery.addSelect('cpu.status as contentPreferenceStatus');
+        searchQuery.leftJoin(
+          ContentPreferenceUser,
+          'cpu',
+          'cpu.referenceId = u.id AND cpu.type = :cpuType AND cpu.userId = :cpuUserId',
+          {
+            cpuType: ContentPreferenceType.User,
+            cpuUserId: ctx.userId,
+          },
+        );
```

Comment:
Yeah this sounds good, I was thinking if we maybe also want to expose usual `ContentPreference` columns instead of just status, its basically the same since you already join/select?

Reply:
so:
- createdAt
- referenceId
- type

Reply:
Just to keep it inline with what frontend types expect.

Reply:
Hmm we can try? would be ideal I guess if it's just a extendable entity.
Let me try some magic